packages/jest-circus/src/__mocks__/test_utils.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @flow
+ */
+
+'use strict';
+
+import os from 'os';
+import path from 'path';
+import {spawnSync} from 'child_process';
+import fs from 'fs';
+
+const CIRCUS_PATH = path.resolve(__dirname, '../../build/index');
+const CIRCUS_RUN_PATH = path.resolve(__dirname, '../../build/run');
+const CIRCUS_STATE_PATH = path.resolve(__dirname, '../../build/state');
+const TEST_EVENT_HANDLER_PATH = path.resolve(__dirname, './test_event_handler');
+
+const runTest = (source: string) => {
+  const tmpFilename = path.join(os.tmpdir(), 'circus-test-file.js');
```

Comment:
this is the safest, but not the most pleasant way of writing files.
I was thinking about creating a gitignored directory (`files_for_integration_tests`) and write everything there, so it's easy to cd into the dir and see what's up in there